Index: src/main/java/com/example/spring_project1/community/controller/ViewController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/spring_project1/community/controller/ViewController.java b/src/main/java/com/example/spring_project1/community/controller/ViewController.java
--- a/src/main/java/com/example/spring_project1/community/controller/ViewController.java	
+++ b/src/main/java/com/example/spring_project1/community/controller/ViewController.java	(date 1714587460485)
@@ -80,7 +80,7 @@
     }
 
     @PostMapping("/edit/{boardid}")
-    public String editBoard(@PathVariable long boardid, @RequestParam String title, @RequestParam String pw, @RequestParam String sub_title) {
+    public String editBoard(@PathVariable("boardid") long boardid, @RequestParam String title, @RequestParam String pw, @RequestParam String sub_title) {
         LocalDateTime createdAt = boardService.findBoard(boardid).getCreatedAt();
         BoardUpdateDto boardUpdateDto = new BoardUpdateDto(boardid, pw, title, sub_title, createdAt, LocalDateTime.now());
         boardService.updateBoard(boardUpdateDto.toEntity());
